### PR TITLE
elevator subsystem

### PR DESCRIPTION
**Pull request by helenaheinonen**
wrote elevator code: moves grabber up and down to reach cubes

### CODE REVIEW

**Review by Hannah3333 — APPROVED** [review]
LAAGASTM

**Review by porglezomp — CHANGES_REQUESTED** [review]
This looks good, but we should get some tests on this. Merge from develop into your branch, and write a test in the tests folder to make sure these do what we expect.

File: tests/elevator_test.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+def test_elevator():
+    motor = GetSet(0)
+    elevator = Elevator(motor)
+
+    elevator.go_up()
+    assert motor.state == 1
+    elevator.go_up(speed=0.5)
+    assert motor.state == 0.5
+
```
Comment (porglezomp) [inline tests/elevator_test.py:22]: nice separation

**Review by porglezomp — APPROVED** [review]
This looks good! 😺

**Pull request merged**